### PR TITLE
Completed command-line-basics exercise

### PR DESCRIPTION
**Pull request by BlakexLim**
cmd line basics

### CODE REVIEW

File: command-line-basics/command-line-basics-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  - `mkdir`
+
+  - `mv`
+
+  - `rm`
+
+  - `cp`
+
+- What are the three virtues of a great programmer?
```
Comment (shawnkost) [inline command-line-basics/command-line-basics-quiz-notes.md:33]: Don't forget about the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Need to answer the quiz questions

File: command-line-basics/command-line-basics-quiz-notes.md (hunk 1 above)
Comment (BlakexLim) [reply command-line-basics/command-line-basics-quiz-notes.md:33]: noooo they didn't save

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**